### PR TITLE
components/refactor total display

### PR DESCRIPTION
**Pull request by marisamodolo**
## Contexto
Alguns ajustes visuais são necessários no componente TotalDisplay devido a troca da fonte básica do projeto para Rubik.
Este PR:
- Padroniza o tamanho da fonte do componente para `large`.
- Alinha o simbolo de `R$` com o valor.

## Checklist
- [x] O tamanho da fonte do texto principal deve passar de 28px para 24px
- [x] A cor do texto principal deve ser #4d4f62.
- [x] A altura da div ao redor do título deve ser removida para que ela cresça de acordo com o texto contido. Essa `div` fica dentro do componente `DataDisplay`, que está dentro de `TotalDisplay`.

## Issues linkadas
- [x] Resolves #https://github.com/pagarme/pilot/issues/1312

### Layout:
image

### Preview:
totaldisplaylarge

## Como testar?
- Entre em pilot/ e rode `yarn` para baixar as dependências.
-  Entre em pilot/packages/pilot e rode ``yarn storybook``.

### As alterações podem ser checadas: 
- Nos componentes:
BalanceSumary, Summary e TotalDisplay.
- Em pages containers:
Anticipation: Form, Confirmation e Result.
Withdraw: Form, Confirmation e Result.
-  Na page:
TransactionDetails.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d2e6855f73c38093763f0d76ec8fd7ba4159ec35
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d2e6855f73c38093763f0d76ec8fd7ba4159ec35
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d2e6855f73c38093763f0d76ec8fd7ba4159ec35
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d2e6855f73c38093763f0d76ec8fd7ba4159ec35
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/components/TotalDisplay/style.css [hunk 1]
```diff
@@ -21,7 +21,7 @@
   }
 
   & span {
-    color: var(--color-light-steel-100);
+    color: var(--color-light-noir-100);
```
Comment (vagnervst) [inline packages/pilot/src/components/TotalDisplay/style.css]: No console estão aparecendo warnings sobre esta cor não existir, acho que vale fazermos o bump do `former-kit-skin-pagarme` para a versão `1.5.1`, onde essa cor foi adicionada 🙂

Comment (miglsoares) [reply packages/pilot/src/components/TotalDisplay/style.css]: https://github.com/pagarme/pilot/blob/release/v0.21.0/packages/pilot/package.json#L13

Comment (vagnervst) [reply packages/pilot/src/components/TotalDisplay/style.css]: estranho 🤔 acho que era algo em minha `node_modules`

**Pull request merged**